distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalogValidation.java

```diff
@@ -76,149 +69,129 @@
+      configureMetacardValidityFilterPlugin(
+          Collections.singletonList("invalid-state=guest"), false, false, getAdminConfig());
     } catch (Exception e) {
       LoggingUtils.failWithThrowableStacktrace(e, "Failed in @BeforeExam: ");
     }
   }
 
   @Before
-  public void setup() throws Exception {
-    configureEnforceValidityErrorsAndWarnings("false", "false", getAdminConfig());
-    configureEnforcedMetacardValidators(Collections.singletonList(""), getAdminConfig());
-    configureFilterInvalidMetacards("false", "false", getAdminConfig());
-    configureMetacardValidityFilterPlugin(
-        Collections.singletonList("invalid-state=guest"), getAdminConfig());
+  public void setup() {
     clearCatalogAndWait();
   }
 
+  /* ***************** TEST ENFORCE VALIDATION ON INGEST ***************** */
   @Test
   public void testEnforceValidityErrorsOnly() throws Exception {
-    // Configure to enforce validator
-    configureEnforcedMetacardValidators(
-        Collections.singletonList(SAMPLE_VALIDATOR), getAdminConfig());
+    // Configure marker plugin to enforce validator errors but not warnings
+    Dictionary<String, Object> markerPluginProps =
+        configureValidationMarkerPlugin(
+            Collections.singletonList(SAMPLE_VALIDATOR), true, false, getAdminConfig());
 
-    // Configure to enforce errors but not warnings
-    configureEnforceValidityErrorsAndWarnings("true", "false", getAdminConfig());
-
-    ingestXmlFromResourceAndWait(WARNING_METACARD);
-    ingestXmlFromResourceAndWait(CLEAN_METACARD);
+    String warningId = ingestXmlFromResourceAndWait(WARNING_METACARD);
+    String cleanId = ingestXmlFromResourceAndWait(CLEAN_METACARD);
     ingestXmlFromResourceWaitForFailure(ERROR_METACARD);
 
-    configureFilterInvalidMetacards("true", "false", getAdminConfig());
+    // verify the clean and warning metacard can be queried
+    query(cleanId, TRANSFORMER_XML, HttpStatus.SC_OK);
+    query(warningId, TRANSFORMER_XML, HttpStatus.SC_OK);
 
-    String query =
-        new CswQueryBuilder().addAttributeFilter(PROPERTY_IS_LIKE, "AnyText", "*").getQuery();
-    ValidatableResponse response =
-        given()
-            .header(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_XML)
-            .body(query)
-            .post(CSW_PATH.getUrl())
-            .then();
+    // Test updating
+    String warningData = getFileContent(WARNING_METACARD);
+    String errorData = getFileContent(ERROR_METACARD);
+    update(cleanId, warningData, MediaType.APPLICATION_XML, HttpStatus.SC_OK);
+    update(cleanId, errorData, MediaType.APPLICATION_XML, HttpStatus.SC_BAD_REQUEST);
 
-    // clean metacard and warning metacard should be in results but not error one
-    response.body(containsString("warning metacard"));
-    response.body(containsString("clean metacard"));
-    response.body(not(containsString("error metacard")));
+    // Reset marker plugin
+    configureValidationMarkerPlugin(markerPluginProps, getAdminConfig());
```

Comment:
❓ How is it reset if the test failed before?

Reply:
I remember when we first started this pattern we were wrapping a majority of the tests in a try/finally. Might go that direction or look into your comment above about making a JUnit Rule for it

Reply:
`try/finally` ... really not a good paradigm for cleaning up in my opinion

Reply:
I'll put it in for now and before working on the bigger suggestions (so it might go away 😄)